docs/en/alab/project_creation.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+comment: no
+header: true
+seotitle: Annotation Lab | John Snow Labs
+title: Project Creation
+permalink: /docs/en/alab/project_creation
+key: docs-training
+modify_date: "2022-10-19"
+use_language_switcher: "Python-Scala"
+show_nav: true
+sidebar:
+    nav: annotation-lab
+---
+
+## New project 
+You can create a new project using a creation wizard which will guide users through each step of the project creation and configuration. Those are illustrated below:
+
+**Project Description**
+
+Every project in Annotation Lab should have the following information:
+
+- a unique name and a short description;
+- a team of annotators, reviewers and a manager which will colaborate on the project;
+- a configuration which specifies the type of annotations that will be created.
+
+To create a new project, click on the Create Project button on the Home Page and choose a name for it. The project can include a short description and annotation instructions/guidelines.
+
+![Description](https://user-images.githubusercontent.com/46840490/193200891-d2098f36-c300-4693-924d-168801ae5acd.png)
+
+Reserved words cannot be used as project names. The use of keywords like count, permission, or name as project names generated UI glitches. To avoid such issues, these keywords are no longer accepted as project names.
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/reserved.gif" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
```

Comment:
You can remove the `align:center;` from the style attribute as it is incorrect property field. And instead of `box-shadow: ....` you can add the class `image__shadow` to the class attribute.